scripts/template/src/patternfly/{folder}/{name}/{nameDasherized}.scss

```
@@ -0,0 +1,26 @@
+// Component variables follow this formula:
+// --{nameBEM}__element--modifier--state--PropertyCamelCase
+
+// The value of component scoped variables is always defined by a global variable:
+.{nameBEM} {
+  // Example block variable
+  --{nameBEM}--Color: var(--pf-global--Color--dark);
```

just want to use more reasonable defaults

```suggestion
  --{nameBEM}--Color: var(--pf-global--Color--100);
```